js/foam/u2/Element.js

```diff
@@ -305,7 +306,49 @@ CLASS({
       name: 'innerHTML',
       transient: true,
       getter: function() { return this.outputInnerHTML(this.createOutputStream()).toString(); }
-    }
+    },
+    {
+      name: 'minWidth',
```

Comment:
I disagree.  Element's properties are pretty much just virtual DOM properties.  It's not nearly as polluted as View is.  And even if it was a little poluted I don't think that's justification to just keep stuffing properties in here that we find useful in a few contexts.

Having to worry about a trait isn't that bad.  You  still have to set sensible values for these properties if you want to put it in a stack view, and therefore you're pretty much aware of the trait.

If you don't apply the trait and/or don't set sensible values, then the stack view can pick defaults, rather than every element having defaults that probably don't make any sense.